addon/doxysearch/doxyindexer.cpp

```diff
@@ -1,6 +1,7 @@
 /******************************************************************************
  *
  * Copyright (C) 1997-2015 by Dimitri van Heesch.
+ * Copyright (C) 2023      by A. Bolshakov, Renga Software LLC.
```

Comment:
Is there a need to add this copyright line? 
I prefer not to keep track of all contributions in this way.
See also [this article](https://www.linuxfoundation.org/blog/blog/copyright-notices-in-open-source-software-projects)

Reply:
I think, not at all. It just seemed to me that it is one of GPL requirements.
GPL v. 2, p. 2a:
>  You must cause the modified files to carry prominent notices stating that you changed the files and the date of any change.

GPL v. 3, p. 5a:
> The work must carry prominent notices stating that you modified it, and giving a relevant date.